modules/user_data/templates/tfe_vm.sh.tpl

```diff
@@ -21,6 +21,15 @@ case "$release_name" in
 esac
 echo "[Terraform Enterprise] Assuming distribution: '$distribution'" | tee -a $log_pathname
 
+if [[ $distribution == "rhel" ]]
+then
+  echo "[Terraform Enterprise] Patching GCP Yum repo configuration" | tee -a $log_pathname
```

Comment:
This is an interesting element:  yum installs hard failed for rhel instances in us-west1. In order to get builds to fail in that region in new and interesting ways the guidance from Google needed to be followed (link in comments). I don't think this is necessary in us-east4 if history is any indicator *but* given that this handles a toxic situation in at least one region imo it should remain.